packages/user_packages/iotcv2/sampleProp.yml

```diff
@@ -0,0 +1,12 @@
+EdgeSystemName : "EdgeSystem-Name"
```

Comment:
Any reason why we added sampleProp.yml and not reuse sampleProp.conf?

Reply:
I have a example code for toyota simulation where we have some specific info, e.g., for properties list. In addition, this is just examples codes, and wanted to expose to users more options for configurations.

Reply:
This should be moved to conf file keeping it consistent across the codebase.